libs/base/ui/overlays/notification-center/src/notification-center.component.ts

```diff
@@ -23,7 +23,22 @@ export const defaultStrategy: NotificationStrategy = {
 export class NotificationCenterComponent extends LitElement {
   static styles = [notificationCenterBaseStyles];
 
-  @property({ reflect: true }) position?: Positions;
+  @property({ reflect: true }) position?: NotificationPosition;
+
+  @property() set maxWidth(value: string) {
```

Comment:
I didn't get, why do we need these properties? Why not just set these css properies in css files or even is style prop?

I would suggest to define default values for them in oryx-notification-center styles and if need to override them just do this by styles

Reply:
i started like that and have 2nd thoughts. Will give it another go, thx

Reply:
Now I got. But still...
```

protected applyStyles(): string {
   const {maxWidth, marginBlock, marginInline} = this.componentOptions ?? {};
   return `${maxWidth ? `--oryx-notification-max-width: ${maxWidth}`}
   ${maxWidth ? `--oryx-notification-margin-block: ${marginBlock}`}
   ${maxWidth ? `--oryx-notification-margin-inline: ${marginInline}`}
   `.trim() || undefined;
}

<oryx-notification-center
      .position=${this.componentOptions?.position}
      styles=${ifDefined(this.applyStyles())}
      ${ref(this.centerRef)}
    ></oryx-notification-center>`
```